action.yml

```diff
@@ -74,11 +77,19 @@ runs:
   using: composite
   steps:
     - id: imgbuild
-      name: Build and publish Docker image
+      name: Build, publish, and optionally save Docker image as a tar archive.
       uses: docker://public.ecr.aws/l7o7z1g8/actions/kaniko-action:${{ action.scm.sha }}
       with:
         entrypoint: kaniko-action
-        args: --dockerfile "${{ inputs.dockerfile }}" --context "${{ inputs.context }}" --destination "${{ inputs.destination }}" --registry-mirrors "${{ inputs.registry-mirrors }}" --skip-default-registry-fallback="${{ inputs.skip-default-registry-fallback }}" --verbosity "${{ inputs.verbosity }}" --target "${{ inputs.target }}"
+        args: |
+          --dockerfile "${{ inputs.dockerfile }}"
+          --context "${{ inputs.context }}"
+          --destination "${{ inputs.destination }}"
+          --registry-mirrors "${{ inputs.registry-mirrors }}"
+          --skip-default-registry-fallback="${{ inputs.skip-default-registry-fallback }}"
+          --verbosity "${{ inputs.verbosity }}"
+          --target "${{ inputs.target }}"
+          ${{ inputs.tar-path && format('--tarPath {0}', inputs.tar-path) || '' }}
```

Comment:
Please quote the path argument in order to take into account that the path may contain a space: `"{0}"`.

(Also, defining a camel case CLI option name (`--tarPath`) is very unusual. It would be good to follow conventions and see consistency (`--tar-path`) in this matter here but I understand that it is functionally irrelevant.)